src/main/java/Event.java

```diff
@@ -0,0 +1,23 @@
+public class Event extends Task{
+    protected String time;
+
+
+    public Event(String description, String time) {
+        super(description);
+        this.time =time;
+        String echo_msg ="          ____________________________________________________________\n" +
```

Comment:
Spacing should be either on both sides of the equal sign or neither. Having a space only on one side is inconsistent.